.github/workflows/build-and-test.yaml

```diff
@@ -149,10 +149,39 @@ jobs:
         Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
         refreshenv
 
-        choco install make && choco install zip && make build-binaries-windows
+        choco install make && choco install zip && make build-binaries-windows-2019
 
-  buildWindowsDocker:
-    name: Build Windows Docker Images
+  buildWindows2022:
```

Comment:
why do you have separate jobs for each version, have you tried in a single job to run windows on 2019 and 2022? Have you tried using matrix to make it work? You can use some conditions for make targets as well